react_app/src/App.js

```diff
@@ -1,27 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
```

Comment:
Google javascript prefers single quote to double quotes:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Fixed!

Reply:
I only see the changes in a few places. Can you fix for all js files? It is fine to use double quotes in HTML parts but please be consistent in doing that.

Reply:
Fixed!